openadapt/config.py

```diff
@@ -30,10 +30,57 @@
     "RECORD_READ_ACTIVE_ELEMENT_STATE": False,
     # TODO: remove?
     "REPLAY_STRIP_ELEMENT_STATE": True,
+    # IGNORES WARNINGS (PICKLING, ETC.)
+    "IGNORE_WARNINGS": False,
     # ACTION EVENT CONFIGURATIONS
     "ACTION_TEXT_SEP": "-",
     "ACTION_TEXT_NAME_PREFIX": "<",
-    "ACTION_TEXT_NAME_SUFFIX": ">"
+    "ACTION_TEXT_NAME_SUFFIX": ">",
+    # SCRUBBING CONFIGURATIONS
+    "SCRUB_ENABLED": True,
+    "SCRUB_CHAR": "*",
+    "SCRUB_LANGUAGE": "en",
+    "SCRUB_CONFIG_TRF": {
+        "nlp_engine_name": "spacy",
+        "models": [{"lang_code": "en", "model_name": "en_core_web_trf"}],
+    },
+    "DEFAULT_SCRUB_FILL_COLOR": (255, 0, 0),
+    "SCRUB_IGNORE_ENTITIES": [
```

Comment:
While I think we want this to live here, unfortunately I don't think `getenv_fallback` currently supports overriding lists in `.env`. Maybe leave as a `TODO` (or implement if it's not too much trouble!)

Reply:
Yeah, I think a TODO is good for now